libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/ConfigurationCacheForAndroidIT.kt

```diff
@@ -41,6 +41,17 @@ class ConfigurationCacheForAndroidIT : AbstractConfigurationCacheIT() {
         testConfigurationCacheOf(":Lib:compileFlavor1DebugKotlin", ":Android:compileFlavor1DebugKotlin")
     }
 
+    @Test
+    fun testKotlinAndroidProjectWithJetifier() = with(Project("AndroidIncrementalMultiModule")) {
+        applyAndroid40Alpha4KotlinVersionWorkaround()
+        this.projectDir.resolve("gradle.properties").appendText("""
+            
+            android.useAndroidX=true
+            android.enableJetifier=true
```

Comment:
I suspect the bug happens with `android.enableJetifier=false` too, because `IdentityTransform` in AGP also fails with non-existent input file. If so, maybe remove the flag as it will be deprecated at some point?

Reply:
Thanks, let's go with that.